### PR TITLE
Optimize handleObjects method

### PR DESCRIPTION
**Pull request by clyang82**
It seems we do not need to call `getMapping` or `unmarshalFromJSON` for the ObjectDefinition in a loop of relevantNamespaces. Nothing changes in the loop. So move it in the upper level to reduce the cpu and memory consumption.

### CODE REVIEW

File: controllers/configurationpolicy_controller.go [hunk 1]
```diff
@@ -1106,6 +1106,14 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 		numCompliant := 0
 		numNonCompliant := 0
 		handled := false
+		// map raw object to a resource, generate a violation if resource cannot be found
+		mapping, statusUpdateNeeded := r.getMapping(objectT.ObjectDefinition, &plc, indx)
+
+		unstruct, err := unmarshalFromJSON(objectT.ObjectDefinition.Raw)
+		if err != nil {
+			os.Exit(1)
```
Comment (mprahl) [inline controllers/configurationpolicy_controller.go:1114]: I know you didn't write this code but could you please change this to a panic instead?

**Comment by mprahl** [comment]
Nice optimization @clyang82 . Just a minor comment.

**Comment by dhaiducek** [comment]
This a minor but impactful change--thanks, @clyang82!

**Review by mprahl — APPROVED** [review]

**Pull request merged**